pynest/nest/lib/hl_api_simulation.py

```diff
@@ -39,6 +39,49 @@ def Simulate(t):
     sps(float(t))
     sr('ms Simulate')
 
+@check_stack
+def Run(t):
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sps(float(t))
+    sr('ms Run')
+
+@check_stack
+def Prepare():
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sr('Prepare')
+
+
+@check_stack
+def Cleanup():
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sr('Cleanup')
+
+@contextmanager
+def IterateRuns():
+    Prepare()
+    try: yield
+    finally: Cleanup()
```

Comment:
This Python is slightly beyond me ... It seems `yield` yields nothing and I cannot see when this generator would fail to yield anything and thus exit.